test/e2e/package.json

```diff
@@ -10,6 +10,7 @@
 	"main": "greetings.js",
 	"defaultTestArgs": "-g -s $BROWSERSIZE",
 	"scripts": {
+		"build": "tsc --build ./tsconfig.json",
```

Comment:
To keep it consistent with other TS packages in the repo, I'd add

```
		"clean": "tsc --build ./tsconfig.json --clean && npx rimraf dist",
```

Reply:
Will be included in patch to address your comments.